LifeCycleAssessment_Engine/Compute/EvaluateEnvironmentalProductDeclarationByVolume.cs

```diff
@@ -73,10 +74,11 @@ private static EnvironmentalMetricResult EvaluateEnvironmentalProductDeclaration
                 BH.Engine.Base.Compute.RecordError("Volume cannot be calculated from object " + ((IBHoMObject)elementM).BHoM_Guid);
                 return null;
             }
+            ScopeType scope = BH.Engine.LifeCycleAssessment.Query.GetElementScope(elementM);
 
             double quantity = gwpByMaterial.Where(x => !double.IsNaN(x)).Sum();
 
-            return new EnvironmentalMetricResult(((IBHoMObject)elementM).BHoM_Guid, field, 0, ObjectScope.Undefined, ObjectCategory.Undefined, phases, Query.GetElementEpd(elementM), quantity, field);
+            return new EnvironmentalMetricResult(((IBHoMObject)elementM).BHoM_Guid, field, 0, scope, ObjectCategory.Undefined, phases, Query.GetElementEpd(elementM), quantity, field);
```

Comment:
This should be updated to call to the new GetElementEPD method with MaterialComposition included to eliminate the redundant MaterialComposition call within the currently called method.